### PR TITLE
Add documentation for ome/omero-py#418

### PR DESCRIPTION
**Pull request by chris-allan**
Add documentation for ome/omero-py#418

### CODE REVIEW

**Review by sbesson — COMMENTED** [review]
Is it worth updating the`omero.grid.Table` method signatures in this page to mention the optional `current` keyword allowing to specify the Ice context as a dictionary?
For someone who has no experience with the tables API, this would make at least clarify where the `omero.tables.include_row_numbers` is expected to be passed.

As per https://github.com/ome/omero-py/pull/418/files, the `readCoordinates` API should also respect the inclusion flag and could also be annotated with a similar note.

**Comment by will-moore** [comment]
As suggested, an example e.g. `t.slice([0], [0,1], {"omero.tables.include_row_numbers": "false"})` would help clarify the method signature.

**Review by will-moore — APPROVED** [review]
Great, thanks

File: omero/developers/Tables.rst [hunk 1]
```diff
@@ -214,6 +247,23 @@ Main methods
...
+            `omero.tables.include_row_numbers` to `false` in the Ice
+            context passed when you make the call.
+
+            For example:
+
+            ::
+
+                slice(
+                    [0],  # colNumbers
+                    [0, 1]  # rowNumbers
```
Comment (sbesson) [inline omero/developers/Tables.rst]:
```suggestion
                    [0, 1],  # rowNumbers
```

**Review by sbesson — APPROVED** [review]

**Pull request merged**